server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -174,6 +174,11 @@ static TransportVersion def(int id) {
     public static final TransportVersion ML_INFERENCE_IBM_WATSONX_RERANK_ADDED = def(8_840_0_00);
     public static final TransportVersion COHERE_BIT_EMBEDDING_TYPE_SUPPORT_ADDED_BACKPORT_8_X = def(8_840_0_01);
     public static final TransportVersion ELASTICSEARCH_9_0 = def(9_000_0_00);
+    public static final TransportVersion REMOVE_SNAPSHOT_FAILURES_90 = def(9_001_0_01);
```

Comment:
Is there a reason why this one is a patch off of `9_001` but the others are patches from `9_000`?

Reply:
gah, this is a typo.

Reply:
Fixed.